lib/telemetry.js

```diff
@@ -60,5 +66,28 @@ export const startPingLoop = () => {
     point.tag('arch', arch())
     point.tag('deployment_type', DEPLOYMENT_TYPE)
     writeClient.writePoint(point)
-  }, 10 * 60 * 1000)
+    writeClient.flush().catch(handleFlushError)
+    await timers.setTimeout(10 * 60 * 1000) // 10 minutes
+  }
+}
+
+export const runMachinesLoop = async () => {
+  while (true) {
+    const point = new Point('machine')
+    point.intField('cpu_count', cpus().length)
+    point.intField('cpu_speed_mhz', cpus()[0].speed)
+    point.tag(
+      'cpu_brand',
+      /intel/i.test(cpus()[0].model)
+        ? 'intel'
+        : /amd/i.test(cpus()[0].model)
+          ? 'amd'
+          : 'unknown'
+    )
```

Comment:
Please add code to detect ARM, especially Apple Silicon (M1/M2/M3).

Reply:
I've added ARM support for apple silicon. What other CPU brands should we detect?

Reply:
> What other CPU brands should we detect?

IDK 🤷🏻‍♂️ Maybe you can query our current Station `ping` events to see what other platforms we have seen so far?

Reply:
Platform != CPU brand though. Do you think we should submit platform in addition to CPU brand?

Reply:
I don't know what kind of information we want to get out of this telemetry, so it's hard to answer, TBH.

> What other CPU brands should we detect?

What CPU brands are used by home servers like Raspberry PI?

> Do you think we should submit platform in addition to CPU brand?

I think there is no harm in doing that.

Alternatively, instead of submitting `unknown` brand, we can include the platform in that value, e.g. `unknown-x86`, `unknown-x64`, `unknown-arm64`, etc.

Reply:
I don't have access to a Respberry PI, so could only guess :|

> Alternatively, instead of submitting unknown brand, we can include the platform in that value, e.g. unknown-x86, unknown-x64, unknown-arm64, etc.

Good idea! I've improved this even further. Wdyt?